auth/src/main/java/com/firebase/ui/auth/util/data/PrivacyDisclosureUtils.java

```diff
@@ -0,0 +1,86 @@
+package com.firebase.ui.auth.util.data;
+
+import android.content.Context;
+import android.support.annotation.RestrictTo;
+import android.support.annotation.StringRes;
+import android.widget.TextView;
+
+import com.firebase.ui.auth.R;
+import com.firebase.ui.auth.data.model.FlowParameters;
+import com.firebase.ui.auth.util.ui.PreambleHandler;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public class PrivacyDisclosureUtils {
+
+    private static final int NO_TOS_OR_PP = -1;
+
+    public static void setupTermsOfServiceAndPrivacyPolicyText(Context context,
```

Comment:
Wondering if we should shorten this to `setupTosAndPp`. Or at least, `setupTermsOfServiceAndPrivacyPolicyText` is quite a mouthful. 😆 @samtstern Thoughts?

Reply:
I prefer to keep it like this :blush:, do you both want me to change it? @samtstern

Reply:
If you think that's clearer then SGTM 👍.